README.md

```diff
@@ -1,39 +1,39 @@
-# Project Template
+# U of T Monopoly
+Below are the general rules, functionality, and a brief description of the implementation of our Monopoly game. 
+# Game Rules 
+The monopoly game we have implemented is a 2-4 player interactive gameboard program. Upon initialization of the game, users are prompted to role the dice of the program. After a player rolls a dice the following mutually exclusive events may occur: 
+-	A game has 2 - 4 players
+-	Player starts the game with $1500
```

Comment:
A player starts the game with $1500 should mentioned before 'the following mutually exclusive events may occur.' 
'2 - 4 players' is already mentioned before this sentence as well, so it doesn't need to be in the list. 

Append the following to the list:
Land on Pass GO 
  - Collect $200
  
Land on Free Parking / Just Visiting (Jail)
  - Nothing happens
 
Land on Go To Jail 
  - Go to jail, don't pass go (I.e., Do not collect $200)
  
Land on Taxes 
  - Pay $200 or $100

Reply:
The edits have been added